etl/steps/data/garden/un/2023-10-09/plastic_waste.py

```diff
@@ -0,0 +1,151 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from typing import cast
+
+import owid.catalog.processing as pr
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+# Logger
+log = get_logger()
+
+REGIONS = [
+    "North America",
+    "South America",
+    "Europe",
+    "European Union (27)",
+    "Africa",
+    "Asia",
+    "Oceania",
+    "Low-income countries",
+    "Upper-middle-income countries",
+    "Lower-middle-income countries",
+    "High-income countries",
+    "World",
+]
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = cast(Dataset, paths.load_dependency("plastic_waste"))
+
+    # Load regions dataset.
+    ds_regions: Dataset = paths.load_dependency("regions")
+    ds_income_groups = paths.load_dependency("income_groups")
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["plastic_waste"].reset_index()
+
+    #
+    # Process data.
+    #
+
+    # Keep relevant columns
+    log.info("un.comtrade.plastic: keep relevant columns")
+
+    #  Extract relevatn columns
+    #    - Year ("RefYear")
+    #    - Type of flow (export/import, denoted by "flowdesc")
+    #    - Reporting country ("reporterDesc")
+    #    - Partner country ("partnerDesc")
+    #     - Quantity of plastic exported/imported ("cmddesc")
+    #    - Mode of transport ("motDesc")
+    COLUMNS_RELEVANT = ["refyear", "flowdesc", "reporterdesc", "partnerdesc", "qty", "motdesc"]
+    tb = tb[COLUMNS_RELEVANT]
+
+    # Rename year and country column fields
+    log.info("un.comtrade.plastic: rename columns")
+    tb = tb.rename(
+        columns={
+            "refyear": "year",
+            "reporterdesc": "country",
+            "flowdesc": "export_vs_import",
+            "partnerdesc": "partner_country",
+            "motdesc": "mode_of_transport",
+        }
+    )
+
+    # Keep exports/imports to/from World only - could be interesting in the future to create flow diagrams in which case will need to change this
+    tb = tb[tb["partner_country"] == "World"]
+    tb = tb.drop("partner_country", axis=1)
+
+    # Pivot to get exports and imports by each mode of transport
+    tb = pr.pivot(
+        tb, index=["year", "country"], columns=["export_vs_import", "mode_of_transport"], values="qty"
+    ).reset_index()
+    # Rename hiearachical columns after pivoting
+    tb.columns = [f"{col[0]}_{col[1]}" if col[0] not in ["year", "country"] else col[0] for col in tb.columns]
+    # Check that there is no intersection between former and current countries
+    log.info("un.comtrade: handle former countries West Germany and Sudan (former)")
```

Comment:
you can use `paths.log.info` now instead, will print the step name in addition to the name.

```suggestion
    paths.log.info("handle former countries West Germany and Sudan (former)")
```